lib/Dialect/FIRRTL/Transforms/LowerAnnotations.cpp

```diff
@@ -841,6 +841,14 @@ LogicalResult LowerAnnotationsPass::solveWiringProblems(ApplyState &state) {
         return failure();
       }
     }
+    // If wiring using references, check that the sink value we connect to is
+    // passive.
```

Comment:
Can totally support some non-passive sinks (driving all fields individually) in the future, at least as long as it's not a port or something.  Doesn't seem worth supporting.